apps/api/src/app/events/services/sms-service/handlers/bulksms.handler.ts

```diff
@@ -0,0 +1,25 @@
+import { ChannelTypeEnum } from '@novu/shared';
+import { ICredentials } from '@novu/dal';
+import { BulkSmsSmsProvider } from 'D:/Novu_latest/novu/providers/bulkSms/src/lib/bulkSms.provider';
+import { BaseSmsHandler } from './base.handler';
+
+export class BulkSmsSmsHandler extends BaseSmsHandler {
+  constructor() {
+    super('bulksms', ChannelTypeEnum.SMS);
+  }
+  buildProvider(credentials: ICredentials) {
+    const config: {
+      authKey?: string;
+      sender?: string;
+      route?: string;
+      // dltEntityId?: string;
```

Comment:
Is 'dltEntityId' not needed? is so I think we can remove it.

Reply:
Yes, it's a optional parameter we can also remove it.

Reply:
Not sure if we should leave it available or remove it. I had to research what it meant and I found this:
https://www.corefactors.in/blog/what-is-dlt-registration/

From reading it seems it might be mandatory for companies in the future to be able to send SMS to their clients to avoid spam, so that makes me think we should leave it. Also I would add a comment explaining what is for as would help future contributors.

Reply:
We can Keep it but it should not be mandatory. Currently I am using bulksms without that. 
Yes you can add a comment for explaining

Reply:
This hasn't been addressed yet.

Reply:
Please leave the optional paramenter `dltEntityId` as other users might need it and leave a comment what is for, for guidance.

Reply:
@p-fernandez have added optional parameter please check